packages/playground-server/src/methods/token.ts

```diff
@@ -0,0 +1,8 @@
+import { Context } from "koa";
+
+export default function token() {
+  return async function(ctx: Context, next: () => Promise<any>) {
+    // TODO: This should restrict who can request a JSON Web Token.
```

Comment:
I'm happy with the suggestion of rate-limiting tokens.

Reply:
@snario I've reduced the lifespan of a token to 5 seconds in d8d27e7. However, I'm leaving this middleware declared for a bit longer. I think I can add a bit more of security without derailing from the main goal.